tests/e2e/kyma_cli_e2e_test.go

```diff
@@ -0,0 +1,119 @@
+package e2e_test
+
+import (
+	"github.com/kyma-project/cli/internal/cli"
+	. "github.com/kyma-project/cli/tests/e2e"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Kyma Deployment, Enabling and Disabling Module", Ordered, func() {
+	kcpSystemNamespace := "kcp-system"
+	deployments := map[string]string{
+		"template-operator-controller-manager": "template-operator-system",
+		"sample-redis-deployment":              "manifest-redis",
+	}
+	BeforeAll(func() {
```

Comment:
```go
BeforeAll(func() {
    Context("Given a Kyma Cluster", func() {
        It("When `kyma alpha deploy` command is executed")
        Expect(ExecuteKymaDeployCommand()).To(Succeed())

        By("Then the Kyma CR is in a ready state")
        Eventually(IsKymaCRInReadyState).
            WithContext(ctx).
            WithArguments(k8sClient, cli.KymaNameDefault, cli.KymaNamespaceDefault).
            Should(BeTrue())

        By("And the Lifecycle Manager is in a ready state")
        Eventually(IsDeploymentReady).
            WithContext(ctx).
            WithArguments(k8sClient, "lifecycle-manager-controller-manager", kcpSystemNamespace).
            Should(BeTrue())
        })
})
```

Reply:
The "Context" cannot be placed inside the "BeforeAll" node